### PR TITLE
User List Filter 

### PR DESCRIPTION
**Pull request by anuradhas248**
Added filter on Organization and User Role column for User List page .
Story ID- KMS-3020

---
This change is [Reviewable](https://reviewable.io/reviews/opensalt/opensalt/328)

### CODE REVIEW

File: tests/_support/Page/User.php [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 namespace Page;
 
-
 use Behat\Behat\Context\Context;
 use Behat\Gherkin\Node\TableNode;
 use PhpSpec\Exception\Example\PendingException;
```
Comment (JosephLeon) [inline tests/_support/Page/User.php]: Codacy is complaining about short variables here but we actually want to use `$I` as it makes sense when reading the code like gherkin style. The way to disable the codacy test here is to add this before the class:

```
/**
 * @SuppressWarnings(PHPMD.ShortVariable)
 */
class User implements Context {
```

File: src/Salt/UserBundle/Repository/OrganizationRepository.php [hunk 2]
```diff
@@ -27,4 +27,20 @@ public function addNewOrganization($organizationName) {
...
+    /**
+     * Search all organizations which contains passed string.
+     *
+     * string|null $search
+     *
+     * @return array
+     */
+    public function findOrg($search): array
+    {
+        $qb = $this->createQueryBuilder('org');
```
Comment (JosephLeon) [inline src/Salt/UserBundle/Repository/OrganizationRepository.php]: Codacy was complaining about the variable length, maybe replace `$qb` with `$query`.

File: src/Salt/UserBundle/Form/Type/SearchForm.php [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+ */
+class SearchForm extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('organization', TextType::class, [
```
Comment (JosephLeon) [inline src/Salt/UserBundle/Form/Type/SearchForm.php]: Should be:

```
->add('organization', TextType::class, [
  'label' => 'Organization',
  'required' => false,
])
```

Try and make sure we don't have wrong indentation or extra spaces anywhere.

Also check lines 25-34 for the same issue.

**Comment by anuradhas248** [comment]
@JosephLeon - I have taken care of your comments. Please verify and confirm.

**Comment by JorgeLu1s** [comment]
@anuradhas248 Since you are implementing jquery datatables now, there is no need to keep the previous code, because it won't be used. I strongly suggest to close this PR, delete this branch and create a new branch with only the `$("#user_table").DataTables();` code, also would be better to add a `datatable` class to the html table and ask for that class in the javascript: `$('.datatable').DataTables();` so in the future we don't have to modify the javascript, just have add to add a class to the html table ;)

**Comment by JorgeLu1s** [comment]
@anuradhas248 Also you have code related to the comments feature in this branch, you need to move this code to a different one and create a new PR for that.

**Pull request closed without merging**